src/file/File.php

```diff
@@ -67,6 +67,27 @@ public function write(Stringable|string $contents): self {
 		return $this;
 	}
 
+	/**
+	 * Append the content at the end of the file.
+	 *
+	 * @param Stringable|string $contents The content to append
+	 *
+	 * @throws FileException if the file does not exist or is not readable
```

Comment:
```suggestion
	 * @throws FileException if the file does not exist or is not writable
```

I guess you mean writable here?